### DISCUSSION TITLE
Cannot wake device

### DISCUSSION THREAD

**Discussion by KopyTKG** [seriousm4x/UpSnap#351] [Help wanted] [answered]
Hello,
I've deployed UpSnap with portainer and docker. Setting the container to run in privilaged mode and device still doesn't turn on.
I've used scan to add device and manualy checked the MAC address to make sure it is correct. I dont really know where is the issue.

**Comment by seriousm4x** [comment]
Are you using host network mode?
Also, what OS is the computer running that you want to wake? Windows can be quite tricky.

**Reply by KopyTKG** [reply]
Yes i am using host network.
![2023-12-05_20-35](https://github.com/seriousm4x/UpSnap/assets/47574331/04185f51-109c-407f-9f42-7c52ee0e992a)
And the PC is running Arch linux. WOL is working perfectly if i use etherwake or wakeonlan cli commands

PC  - ethertool
![2023-12-05_20-46](https://github.com/seriousm4x/UpSnap/assets/47574331/62548837-c81f-49b1-953f-1106f08b2acd)

**Reply by KopyTKG** [reply]
Could there be a problem with VLANs?

**Reply by seriousm4x** [reply, marked as answer]
Yes, if you use vlans you need to make sure the netmask address is set properly. You could just set it to `255.255.255.255` to send the magic packet to all vlans.

**Reply by KopyTKG** [reply]
> Yes, if you use vlans you need to make sure the netmask address is set properly. You could just set it to `255.255.255.255` to send the magic packet to all vlans.

Thanks
